frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/patient-popup/patient-popup.component.html

```diff
@@ -26,84 +26,186 @@ <h5 class="popover-header">
-                        <small>
-                            ({{ patient.personalInformation.birthdate }})
-                        </small>
-                    </td>
-                </tr>
-                <tr>
-                    <th scope="col">Geschlecht</th>
-                    <td
-                        *appLet="patient.biometricInformation.sex as sex"
-                        class="font-monospace"
-                    >
-                        {{
-                            sex === 'male'
-                                ? 'männlich'
-                                : sex === 'female'
-                                ? 'weiblich'
-                                : 'unbestimmt'
-                        }}
-                    </td>
-                </tr>
-                <tr>
-                    <th scope="col">Biometrie</th>
-                    <td class="font-monospace">
-                        {{ patient.biometricInformation.externalFeatures }}
-                    </td>
-                </tr>
-                <tr>
-                    <th scope="col">Adresse</th>
-                    <td class="font-monospace">
-                        {{ patient.personalInformation.address }}
-                    </td>
-                </tr>
-                <ng-container
-                    *ngIf="(apiService.currentRole$ | async) !== 'participant'"
-                >
-                    <tr>
-                        <th scope="col">Beschreibung</th>
-                        <td class="font-monospace">
-                            <app-patient-status-display
-                                [patientStatus]="patient.patientStatusCode"
+                                <tr>
+                                    <th scope="col">Beschreibung</th>
+                                    <td class="font-monospace fs-5">
+                                        <app-patient-status-display
+                                            [patientStatus]="
+                                                patient.patientStatusCode
+                                            "
+                                        >
+                                        </app-patient-status-display>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <th scope="col">
+                                        Verletzungs-Geschwindigkeit:
+                                    </th>
+                                    <td class="font-monospace">
+                                        &times;{{ patient.timeSpeed }}
+                                    </td>
+                                </tr>
+                            </ng-container>
+                        </tbody>
+                    </table>
+                </ng-template>
+            </ng-container>
+            <ng-container ngbNavItem>
+                <a ngbNavLink>Vorsichtung</a>
```

Comment:
Maybe rename this link when the pretriage is disabled, as this may lead to confusion.

Reply:
@Dassderdie Do you approve the solution of https://github.com/hpi-sam/digital-fuesim-manv/pull/419/commits/2754765c52d6b1ef6803c4a6bb9cd8981b9cec3a?